templates/ts/copilot-plugin-from-scratch/appPackage/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.15",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com/privacy",
+        "termsOfUseUrl": "https://www.example.com/termsofuse"
+    },
+    "icons": {
+        "color": "color.png",
+        "outline": "outline.png"
+    },
+    "name": {
+        "short": "copilot-plugin-tpl-${{TEAMSFX_ENV}}",
```

Comment:
I wonder if we should use "template" or just remove "tpl" in this project. since others may not know what it stands for.

Reply:
I think the app name should be a placeholder {{appName}}

Reply:
Thanks for reminding, I'll fix this problem.